.circleci/config.yml

```diff
@@ -106,9 +107,11 @@ jobs:
       - setup_integration_test
       - save_test_results
   integration-tests:
+    parallelism: 4
     executor: integration_test_exec
     environment:
       TESTING_PROFILE: integration-tests
+      TEST_LIST: integration-tests.txt
```

Comment:
amused by the naming scheme, I guess if this needed to match the others it would need to be IT.txt. Maybe they should all just be a folder

Reply:
Admittedly, the naming scheme was chosen when I was trying to run language parsing tests in parallel (unfortunately didn't work out) because language parsing tests had both unit test and integration tests, so I was trying to make it obvious from the name what type of test each file contained.

Reply:
What do you mean by folder? As in move all the tests in `dockstore-integration-testing/src/test/java/io/dockstore/client/cli` that are part of `integration-tests` to a new file called `dockstore-integration-testing/src/test/java/io/dockstore/client/cli/IT`?

Reply:
Oh no, I just meant all the temp files have a suffix of `IT.java` except for `integration-tests.txt`
So maybe you could create folder called `IT` and put all the files in there and remove the suffix. I was just musing, it's not important.

Reply:
I have changed the file names